src/lib.rs

```diff
@@ -110,6 +77,57 @@ extern crate serde_derive;
 #[macro_use]
 mod macros;
 
+// ############################################################################
+// Public API
+// ############################################################################
+pub use self::{
+    chain::quorum_count,
+    error::{InterfaceError, RoutingError},
+    event::{ClientEvent, ConnectEvent, Event},
```

Comment:
Not sure ConnectEvent is required in API as of yet either

Reply:
It's part of `Event` which is public, so probably should be public too.